pkgs/servers/monitoring/prometheus/promscale/default.nix

```diff
@@ -29,6 +29,10 @@ buildGoModule rec {
     fi
   '';
```

Comment:
Would be nice to also remove `doInstallCheck` and `installCheckPhase` and move to `testVersion` in this update:
```diff
 { lib
 , buildGoModule
 , fetchFromGitHub
+, promscale
+, testVersion
 }:

-  doInstallCheck = true;
-  installCheckPhase = ''
-    if [[ "$("$out/bin/${pname}" -version)" == "${version}" ]]; then
-      echo '${pname} smoke check passed'
-    else
-      echo '${pname} smoke check failed'
-      exit 1
-    fi
-  '';

+  passthru.tests.version = {
+    package = promscale;
+    command = "promscale -version";
+  };

```

Reply:
👍